provider-springmvc/src/main/java/io/servicecomb/provider/springmvc/reference/RestTemplateWrapper.java

```diff
@@ -27,26 +29,43 @@
 import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
 
-import io.servicecomb.common.rest.RestConst;
-
 /**
  * 用于同时支持cse调用和非cse调用
  */
 public class RestTemplateWrapper extends RestTemplate {
-    private static RestTemplate cseRestTemplate = new CseRestTemplate();
+    private static List<AcceptableRestTemplate> acceptableRestTemplates = new ArrayList<>();
```

Comment:
a better design is to declare default rest template and avoid null checking all together
```
class RestTemplateWrapper implements RestOperations {
  private static final RestTemplate defaultTemplate =  new RestTemplate();
  private static List<AcceptableRestTemplate> acceptableRestTemplates = asList(new CseRestTemplate());

    private RestTemplate getRestTemplate(String url) {
        for (AcceptableRestTemplate template : acceptableRestTemplates) {
            if (template.isAcceptable(url)) {
                return template;
            }
        }
        return defaultTemplate;
    }
}
```